src/js/components/Alert.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const Alert = ({ type = 'info', heading, text, htmlElement }) => {
+const Alert = ({ type = 'info', heading, text, children }) => {
   if (!text) return null
 
   return (
     <div className={`usa-alert usa-alert-${type}`}>
       <div className="usa-alert-body">
         {heading ? <h3 className="usa-alert-heading">{heading}</h3> : null}
         <p className="usa-alert-text">{text}</p>
-        {htmlElement ? htmlElement : null} {/* allow for additional html */}
+        {children ? children : null}
```

Comment:
afaik `props.children` will always either be a valid element or `null`, so you don't need this check here